sites/docs/src/IMAGE_RENDERING.md

```diff
@@ -24,7 +24,7 @@ To tackle this **_we look to [WebGL](https://developer.mozilla.org/en-US/docs/We
 
 ###### 2. Serving Data Efficiently
 
-Most high resolution tiling image viewing systems rely on `jpeg` files, which can obtain 10:1 compression ratios with imperceptible loss of quality, to serve the image tiles. The `jpeg` files are very small and therefore there is minimal bottleneck with network requests. However, this file type/compression scheme is [not an easy option](https://caniuse.com/#feat=jpegxr) at this moment for more than 8 bits of data per channel in current browsers as they do not ship with the decompression algorithm for more than 8 bit data. However libraries like [zarr.js](https://github.com/gzuidhof/zarr.js) and [geotiff.js](https://github.com/geotiffjs/geotiff.js) could eventually use [an implementation](https://github.com/mozilla/pdf.js/blob/fa4b431091c51a82315dce7da7b848cc6498bcea/src/core/jpg.js) to decode said data. It also it clear that we wish to lose any precision in what is essentially a scientific measurement. The other popular image file type, `png`, achieves considerably worse compression (around 2:1) but is lossless.
+Most high resolution tiling image viewing systems rely on `jpeg` files, which can obtain 10:1 compression ratios with imperceptible loss of quality, to serve the image tiles. The `jpeg` files are very small and therefore there is minimal bottleneck with network requests. However, this file type/compression scheme is [not an easy option](https://caniuse.com/#feat=jpegxr) at this moment for more than 8 bits of data per channel in current browsers as they do not ship with the decompression algorithm for more than 8 bit data. However libraries like [zarr.js](https://github.com/gzuidhof/zarr.js) and [geotiff.js](https://github.com/geotiffjs/geotiff.js) could eventually use [an implementation](https://github.com/mozilla/pdf.js/blob/fa4b431091c51a82315dce7da7b848cc6498bcea/src/core/jpg.js) to decode said data. It is also it clear that we wish to lose any precision in what is essentially a scientific measurement. The other popular image file type, `png`, achieves considerably worse compression (around 2:1) but is lossless.
```

Comment:
```suggestion
Most high resolution tiling image viewing systems rely on `jpeg` files, which can obtain 10:1 compression ratios with imperceptible loss of quality, to serve the image tiles. The `jpeg` files are very small and therefore there is minimal bottleneck with network requests. However, this file type/compression scheme is [not an easy option](https://caniuse.com/#feat=jpegxr) at this moment for more than 8 bits of data per channel in current browsers as they do not ship with the decompression algorithm for more than 8 bit data. However libraries like [zarr.js](https://github.com/gzuidhof/zarr.js) and [geotiff.js](https://github.com/geotiffjs/geotiff.js) could eventually use [an implementation](https://github.com/mozilla/pdf.js/blob/fa4b431091c51a82315dce7da7b848cc6498bcea/src/core/jpg.js) to decode said data. It is also clear that we wish to lose any precision in what is essentially a scientific measurement. The other popular image file type, `png`, achieves considerably worse compression (around 2:1) but is lossless.
```

Reply:
extra "it" in your fix :)

Reply:
oh god 🤦

Reply:
Ok sorry to do this to you, but also it should probably say "It is also _not_ clear..." since the whole points is that you don't probably don't want to lose information in a scientific setting.....sorry!

Reply:
Okay so I keep re-reading this paragraph and I think the wording is just confusing. The phrase "It is not clear that we wish to lose" is so full of double negatives it's almost impossible to figure out what's actually meant without reading it three times. Can we write instead:
> That said, we may not want to lost any precision whatsoever in what is essentially a scientific measurement, so compression methods like JPEG where any information is lost may be a non-starter.

Or something like this? Am I capturing the point correctly?

Reply:
Yes! Perfecto!  I am notoriously bad at writing, so really any help is welcome :)